### PR TITLE
Convert a bunch of HTTP links to HTTPS

### PR DESCRIPTION
**Pull request by strugee**
What the title says.

I stopped halfway through doing this and started writing a tool to automate changes like this, but since that was several months ago and the changes that _are_ there are still sitting on my disk unmerged, I figured I'd send this in the meantime.

### CODE REVIEW

File: app/models/agents/stubhub_agent.rb [hunk 1]
```diff
@@ -63,7 +63,7 @@ def event_id
       end
 
       def base_url
-       'http://www.stubhub.com/listingCatalog/select/?q='
+       'https://www.stubhub.com/listingCatalog/select/?q='
```
Comment (dsander) [inline app/models/agents/stubhub_agent.rb]: I think you also need to adjust the specs to match the https protocol.

Comment (strugee) [inline app/models/agents/stubhub_agent.rb]: @dsander fixed! :)

**Comment by cantino** [comment]
Great idea, thanks!

**Comment by strugee** [comment]
@cantino @dsander some of the Stubhub specs are still failing; you should probably revert this merge

**Comment by strugee** [comment]
Sent a followup PR which fixed tests.

**Comment by cantino** [comment]
Whoops, thanks!

**Pull request merged**